MonoGame.Framework/Windows/WinFormsGameForm.cs

```diff
@@ -137,5 +153,15 @@ protected override void WndProc(ref Message m)
 
             base.WndProc(ref m);
         }
+
+        private static ushort HiWord(IntPtr word)
+        {
+            return (ushort)(((ulong)word >> 16) & 0xffff);
+        }
+
+        private static short GetWheelDeltaWParam(IntPtr wParam)
+        {
+            return (short)HiWord(wParam);
```

Comment:
If it is only used once, and it is a simple cast of another return value, just put this inline in the call site.  Adding it as a separate method is overkill.